docs/api/other-api/node.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "@react-router/node"
+---
+
+# `@react-router/node`
+
+This package contains utilities and polyfills for Node.js.
+
+## Version Support
```

Comment:
This doc is a bit light since we no longer need the `Polyfills` section, but the `Version Support` is important to have documented somewhere.  I'm open to suggestions if there's an existing doc it could be better suited in to avoid a new sidebar entry just for this?

Reply:
The line above states

> This package contains utilities and polyfills for Node.js.

Is the "pollyfills" part no longer true then?

My only other suggestion for this file would be to link to the reference docs I suppose: https://api.reactrouter.com/v7/modules/_react_router_node.html

Reply:
oh good catch - and yeah reference docs is a good idea.  I think I might actually collapse this down into `{adapters}` since this is related to that and can be used to just wire it up with raw node `http`.  And then I'll also add the reference links for all the adapters in there.